airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -38,41 +38,63 @@
 import io.airbyte.scheduler.models.IntegrationLauncherConfig;
 import io.airbyte.scheduler.models.JobRunConfig;
 import io.temporal.client.WorkflowClient;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
 
 public class TemporalClient {
 
   private final WorkflowClient client;
 
+  public static TemporalClient production() {
+    return new TemporalClient(TemporalUtils.TEMPORAL_CLIENT);
+  }
+
   public TemporalClient(WorkflowClient client) {
     this.client = client;
   }
 
-  public ConnectorSpecification submitGetSpec(long jobId, int attempt, JobGetSpecConfig config) throws TemporalJobException {
+  public ConnectorSpecification submitGetSpec(UUID jobId, int attempt, JobGetSpecConfig config) throws TemporalJobException {
     final JobRunConfig jobRunConfig = TemporalUtils.createJobRunConfig(jobId, attempt);
 
     final IntegrationLauncherConfig launcherConfig = new IntegrationLauncherConfig()
-        .withJobId(jobId)
+        .withJobId(jobId.toString())
         .withAttemptId((long) attempt)
         .withDockerImage(config.getDockerImage());
     return getWorkflowStub(SpecWorkflow.class, TemporalJobType.GET_SPEC).run(jobRunConfig, launcherConfig);
 
   }
 
-  public StandardCheckConnectionOutput submitCheckConnection(long jobId, int attempt, JobCheckConnectionConfig config) throws TemporalJobException {
+  public StandardCheckConnectionOutput submitCheckConnection(UUID jobId, int attempt, JobCheckConnectionConfig config) throws TemporalJobException {
     final JobRunConfig jobRunConfig = TemporalUtils.createJobRunConfig(jobId, attempt);
     final IntegrationLauncherConfig launcherConfig = new IntegrationLauncherConfig()
-        .withJobId(jobId)
+        .withJobId(jobId.toString())
         .withAttemptId((long) attempt)
         .withDockerImage(config.getDockerImage());
     final StandardCheckConnectionInput input = new StandardCheckConnectionInput().withConnectionConfiguration(config.getConnectionConfiguration());
 
     return getWorkflowStub(CheckConnectionWorkflow.class, TemporalJobType.CHECK_CONNECTION).run(jobRunConfig, launcherConfig, input);
   }
 
-  public AirbyteCatalog submitDiscoverSchema(long jobId, int attempt, JobDiscoverCatalogConfig config) throws TemporalJobException {
+  private static <T> T execute2(CompletableFuture<T> completableFuture) throws TemporalJobException {
```

Comment:
execute2? I am sure there is a better naming for this.

Reply:
bah. oops. that was supposed to be deleted entirely. my bad.